### ISSUE TITLE
How can I sent device information from 3rd party app to edgecore by http or https protocol?

### ISSUE THREAD

**Issue by 15506677618** [kubeedge/kubeedge#5972]
**What happened and what you expected to happen**:

Hi team,
one question, I am using the v1.19 version kubeedge, I want to update device data directlly at edge side, not by cloud side api, means, I want to post data to edge node by some customerize restful api, how can I do that? I think mapper-framework, maybe can do this, can you give one demo or any useful docs? any good suggestion?

**Anything else we need to know?**:
I try to follow below doc to set the http protocol, but not worked
![mapper](https://github.com/user-attachments/assets/d89d1f72-dd83-424e-9e1b-a29a8799859c)

**Environment**:
- Kubernetes version (v1.29.10):
- KubeEdge version(1.19):

**Comment by wbc6080** [comment]
Mapper is a device management plug-in in kubeedge, which can manage edge IoT devices to kubeedge cluster. Is your need to transmit data from edge IoT devices to your own application through http?

**Comment by 15506677618** [comment]
> Mapper is a device management plug-in in kubeedge, which can manage edge IoT devices to kubeedge cluster. Is your need to transmit data from edge IoT devices to your own application through http?

No, my requirement is transmit data from our own application to EdgeCore through http/https. as we have some logic already here, so can 3rd app post data to EdgeCore by http/https, I am look for such solution.

**Comment by wbc6080** [comment]
v1.19 of kubeedge supports two ways to transfer data to IoT devices. The first is by updating the device property value directly in the device instance in the cloud; the other is through the device data write interface provided by the edge mapper. The second is our latest feature implementation in v1.19, which allows users to create device data write requests through `http`. You can refer to https://deploy-preview-646--kubeedge.netlify.app/docs/developer/mapper-framework#device-write
https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/mapper-support-devicedata-writing.md
 to see if it meets your needs.
